test/images/httpd-new/BASEIMAGE

```diff
@@ -3,5 +3,6 @@ linux/arm=arm32v6/httpd:2.4.39-alpine
 linux/arm64=arm64v8/httpd:2.4.39-alpine
 linux/ppc64le=ppc64le/httpd:2.4.39-alpine
 linux/s390x=s390x/httpd:2.4.39-alpine
-windows/amd64/1809=REGISTRY/busybox:1.29-2-windows-amd64-1809
-windows/amd64/ltsc2022=REGISTRY/busybox:1.29-2-windows-amd64-ltsc2022
+windows/amd64/1809=REGISTRY/busybox:1.29-5-windows-amd64-1809
+windows/amd64/ltsc2019=REGISTRY/busybox:1.29-5-windows-amd64-1809
```

Comment:
same here?

Reply:
I think there's no ltsc2019 busybox image so use 1809 instead. Do you think it's the correct way?

Reply:
ah I see, yes that will work. I don't know that we need the 1809 tags for the tests images but I don't think it hurts since it doesn't cost more to have an extra tag.